test/gurka/test_conditional_restrictions.cc

```diff
@@ -30,37 +30,45 @@ class ConditionalRestrictions : public ::testing::Test {
 
     const std::string timeDenied =
         "no @ (Mar 00:00-07:00;Mar 17:30-24:00;Apr 00:00-07:00;Apr 19:00-24:00;Aug 00:00-07:00;Aug 19:00-24:00)";
-    const std::string timeAllowed =
+    const std::string yesAllowed =
         "yes @ (Mar 00:00-07:00;Mar 17:30-24:00;Apr 00:00-07:00;Apr 19:00-24:00;Aug 00:00-07:00;Aug 19:00-24:00)";
+    const std::string privateAllowed =
+        "private @ (Mar 00:00-07:00;Mar 17:30-24:00;Apr 00:00-07:00;Apr 19:00-24:00;Aug 00:00-07:00;Aug 19:00-24:00)";
+    const std::string deliveryAllowed =
+        "delivery @ (Mar 00:00-07:00;Mar 17:30-24:00;Apr 00:00-07:00;Apr 19:00-24:00;Aug 00:00-07:00;Aug 19:00-24:00)";
+    const std::string designatedAllowed =
+        "delivery @ (Mar 00:00-07:00;Mar 17:30-24:00;Apr 00:00-07:00;Apr 19:00-24:00;Aug 00:00-07:00;Aug 19:00-24:00)";
+    const std::string destinationAllowed =
+        "delivery @ (Mar 00:00-07:00;Mar 17:30-24:00;Apr 00:00-07:00;Apr 19:00-24:00;Aug 00:00-07:00;Aug 19:00-24:00)";
     const gurka::ways ways = {
         {"AD",
          {{"highway", "service"},
           {"motorcar", "no"},
           {"bicycle", "yes"},
           {"foot", "yes"},
-          {"bicycle:conditional", timeDenied},
-          {"foot:conditional", timeAllowed}}},
+          {"bicycle:conditional", yesAllowed},
```

Comment:
The restriction changed from "denied" to "allowed" but none of the tests below needed to be changed? So for the time it was denied would need to have its test changed, no?

Reply:
@mandeepsandhu before we only tested for the `yes` attribute.  So I added the others in randomly for the tests regardless of the mode as it hits the same code.  If we don't encounter one of those values we default to denied.

Reply:
Got it. thx